hnn_core/cell.py

```diff
@@ -286,6 +295,11 @@ class Cell:
     sect_loc : dict of list
         Can have keys 'proximal' or 'distal' each containing
         names of section locations that are proximal or distal.
+    cell_tree : dict of list
+        Stores the tree representation of a cell. Built from topology.
+        Root is the 0 end of 'soma'. Nodes are a tuple (sec_name, sec_end_pt)
```

Comment:
```suggestion
        Root is the 0 end of 'soma'. Nodes are a tuple (sec_name, node_pos)
```

node_pos = node position

more correct / easier to understand?

Reply:
Should I rename all instances of `sec_end_pt` as `node_pos`?